go-controller/pkg/ovn/namespace.go

```diff
@@ -315,7 +316,16 @@ func (oc *Controller) updateNamespace(old, newer *kapi.Namespace) {
 	aclAnnotation := newer.Annotations[util.AclLoggingAnnotation]
 	oldACLAnnotation := old.Annotations[util.AclLoggingAnnotation]
 	// support for ACL logging update, if new annotation is empty, make sure we propagate new setting
-	if aclAnnotation != oldACLAnnotation && (oc.aclLoggingCanEnable(aclAnnotation, nsInfo) || aclAnnotation == "") {
+	if aclAnnotation != oldACLAnnotation {
+		// In case of an error, log a warning. If the entire annotation could not be parsed, disable logging completely.
+		// If one of the values could not be parsed, disable logging for the respective key.
+		// We do this to achieve idempotency: invalid input should always disable logging instead of relying on the last
+		// valid input.
```

Comment:
this comments looks a bit confusing, because we really disable logging in the `aclLoggingUpdateNsInfo` and not here, here we just log error.
So maybe only leaving `aclLoggingUpdateNsInfo` comment will be enough?

Reply:
Changed to:
~~~
  320         // When input cannot be parsed correctly, aclLoggingUpdateNsInfo disables logging and returns an error. Hence,  
  321         // log a warning to make users aware of issues with the annotation. See aclLoggingUpdateNsInfo for more details.
~~~